### PR TITLE
Add tests for MIME boundary correctness with chunking disabled

### PR DESCRIPTION
**Pull request by Nirothipan**
Refer product-ei/issues/3624

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: integration/mediation-tests/tests-transport/src/test/resources/artifacts/ESB/server/repository/deployment/server/synapse-configs/default/proxy-services/respondBE.xml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+<proxy xmlns="http://ws.apache.org/ns/synapse" name="respondBE"
+       startOnLoad="true"
+       trace="disable" transports="http https">
+    <description/>
+    <target>
+        <inSequence>
+            <log level="full"/>
```
Comment (sdkottegoda) [inline integration/mediation-tests/tests-transport/src/test/resources/artifacts/ESB/server/repository/deployment/server/synapse-configs/default/proxy-services/respondBE.xml]: Would having a log cutom with a unique string be more useful? Also it would be less bulky.

Comment (Nirothipan) [reply integration/mediation-tests/tests-transport/src/test/resources/artifacts/ESB/server/repository/deployment/server/synapse-configs/default/proxy-services/respondBE.xml]: The issue happens when we have log full only when chunking disabled with ESB backend.

Comment (sdkottegoda) [reply integration/mediation-tests/tests-transport/src/test/resources/artifacts/ESB/server/repository/deployment/server/synapse-configs/default/proxy-services/respondBE.xml]: :+1:

**Comment by Nirothipan** [comment]
Please do not merge now .
